pkg/kubectl/cmd/apply/apply_set_last_applied.go

```diff
@@ -128,6 +130,10 @@ func (o *SetLastAppliedOptions) Complete(f cmdutil.Factory, cmd *cobra.Command)
 	if err != nil {
 		return err
 	}
+	o.tenant, o.enforceTenant, err = f.ToRawKubeConfigLoader().Tenant()
```

Comment:
Nit: consider use one lines of if ..., err = ...; err!=nil {

Reply:
All the code to get commandline params in kubectl part does not use one-lines go get err ( like the code right before). So I am not going to be an outliner. ;-)